pkgs/development/python-modules/apache-airflow/default.nix

```diff
@@ -0,0 +1,528 @@
+  # boto3 (<1.8.0,>=1.0.0) ; extra == 'emr'
+  # google-api-python-client (<2.0.0dev,>=1.6.0) ; extra == 'gcp_api'
+  # google-auth-httplib2 (>=0.0.1) ; extra == 'gcp_api'
+  # google-auth (<2.0.0dev,>=1.0.0) ; extra == 'gcp_api'
+  # google-cloud-bigtable (==0.31.0) ; extra == 'gcp_api'
+  # google-cloud-container (>=0.1.1) ; extra == 'gcp_api'
+  # google-cloud-language (>=1.1.1) ; extra == 'gcp_api'
+  # google-cloud-spanner (>=1.7.1) ; extra == 'gcp_api'
+  # google-cloud-translate (>=1.3.3) ; extra == 'gcp_api'
+  # google-cloud-vision (>=0.35.2) ; extra == 'gcp_api'
+  # grpcio-gcp (>=0.2.2) ; extra == 'gcp_api'
+  # httplib2 (~=0.9.2) ; extra == 'gcp_api'
+  # pandas-gbq ; extra == 'gcp_api'
+  # PyOpenSSL ; extra == 'gcp_api'
+  # Flask-OAuthlib (>=0.9.1) ; extra == 'github_enterprise'
+  # oauthlib (!=2.0.3,!=2.0.4,!=2.0.5,<3.0.0,>=1.1.2) ; extra == 'github_enterprise'
+  # requests-oauthlib (==1.1.0) ; extra == 'github_enterprise'
+  # Flask-OAuthlib (>=0.9.1) ; extra == 'google_auth'
+  # oauthlib (!=2.0.3,!=2.0.4,!=2.0.5,<3.0.0,>=1.1.2) ; extra == 'google_auth'
+  # requests-oauthlib (==1.1.0) ; extra == 'google_auth'
+  # snakebite (>=2.7.8) ; extra == 'hdfs'
+  # hmsclient (>=0.1.0) ; extra == 'hive'
+  # pyhive (>=0.6.0) ; extra == 'hive'
+  # jaydebeapi (>=1.1.1) ; extra == 'jdbc'
+  # JIRA (>1.0.7) ; extra == 'jira'
+  # pykerberos (>=1.1.13) ; extra == 'kerberos'
+  # requests-kerberos (>=0.10.0) ; extra == 'kerberos'
+  # thrift-sasl (>=0.2.0) ; extra == 'kerberos'
+  # snakebite[kerberos] (>=2.7.8) ; extra == 'kerberos'
+  # kubernetes (>=3.0.0) ; extra == 'kubernetes'
+  # cryptography (>=2.0.0) ; extra == 'kubernetes'
+  # ldap3 (>=2.5.1) ; extra == 'ldap'
+  # pymongo (>=3.6.0) ; extra == 'mongo'
+  # pymssql (>=2.1.1) ; extra == 'mssql'
+  # mysqlclient (<1.4,>=1.3.6) ; extra == 'mysql'
+  # cx-Oracle (>=5.1.2) ; extra == 'oracle'
+  # bcrypt (>=2.0.0) ; extra == 'password'
+  # flask-bcrypt (>=0.7.1) ; extra == 'password'
+  # pinotdb (==0.1.1) ; extra == 'pinot'
+  # psycopg2 (<2.8,>=2.7.4) ; extra == 'postgres'
+  # qds-sdk (>=1.9.6) ; extra == 'qds'
+  # librabbitmq (>=1.6.1) ; extra == 'rabbitmq'
+  # redis (~=3.2) ; extra == 'redis'
+  # boto3 (<1.8.0,>=1.7.0) ; extra == 's3'
+  # simple-salesforce (>=0.72) ; extra == 'salesforce'
+  # pysmbclient (>=0.1.3) ; extra == 'samba'
+  # analytics-python (>=1.2.9) ; extra == 'segment'
+  # sendgrid (<6,>=5.2.0) ; extra == 'sendgrid'
+  # slackclient (>=1.0.0) ; extra == 'slack'
+  # snowflake-connector-python (>=1.5.2) ; extra == 'snowflake'
+  # snowflake-sqlalchemy (>=1.1.0) ; extra == 'snowflake'
+  # paramiko (>=2.1.1) ; extra == 'ssh'
+  # pysftp (>=0.2.9) ; extra == 'ssh'
+  # sshtunnel (<0.2,>=0.1.4) ; extra == 'ssh'
+  # statsd (<4.0,>=3.0.1) ; extra == 'statsd'
+  # vertica-python (>=0.5.1) ; extra == 'vertica'
+  # hdfs[avro,dataframe,kerberos] (>=2.0.4) ; extra == 'webhdfs'
+  # pywinrm (==0.2.2) ; extra == 'winrm'
```

Comment:
O.o that's a lot of packages.....

you should really only have to concern yourself with whats defined in the setup.py of the apache-airflow package

Reply:
Yeup I'm not going to package any of the extras currently the commits are just the output from `nixpkgs-pytools`

Reply:
I will clean up these commits